modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -1008,4 +1008,4 @@
       }
     }
   }
-}
+}
```

Comment:
I'm not sure what change this is showing here, mine ends on line 1011 and doesn't have a empty space at the end, which appears to be the same as master